js/focuspoint/focuspoint.js

```diff
@@ -120,32 +140,75 @@
 			$imageFocusWrapper.append('<div class="' + css.imageFocus.clickarea + '"></div>');
 		},
 
+
+		moveFocusPointActions: function ()
+		{
+			var $focusPoint = $('.' + css.imageFocus.point);
+
+			$focusPoint.on('mousedown', function ()
+			{
+				focusPointState.move = true;
+
+				//Set current dimension data in case position and size of image are changed because of content changes
+				Plugin.prototype.setImageDimensionData();
+				//Highlight crop button
+				Plugin.prototype.highlightCropButton();
+			});
+
+			$focusPoint.on('mouseup', function ()
+			{
+				focusPointState.move = false;
+			});
+
+			$(window).on('mousemove', function (event)
+			{
+				Plugin.prototype.moveFocusPoint(event);
+			});
+		},
+
 		/**
-		 * Move focus point on img click
+		 * Set focus point on move
 		 */
-		moveFocusPoint: function (event, object)
+		moveFocusPoint: function (event)
 		{
-			var imageW = $(object).width();
-			var imageH = $(object).height();
+			if (focusPointState.move === false) {
+				return false;
+			}
 
 			//Calculate FocusPoint coordinates
-			var offsetX = event.pageX - $(object).offset().left;
-			var offsetY = event.pageY - $(object).offset().top;
-
-			//Calculate CSS Percentages
-			var percentageX = (offsetX / imageW) * 100;
-			var percentageY = (offsetY / imageH) * 100;
+			var offsetX = event.pageX - attachment.position.x;
+			var offsetY = event.pageY - attachment.position.y;
+
+			//Calculate and set percentages
+			attachment.focusPoint.x = (offsetX / attachment.dimension.width) * 100;
+			attachment.focusPoint.y = (offsetY / attachment.dimension.height) * 100;
+
+			if (attachment.focusPoint.x < 0) {
+				attachment.focusPoint.x = 0;
+			} else {
+				if (attachment.focusPoint.x > 100) {
+					attachment.focusPoint.x = 100;
+				}
+			}
 
-			//Write calculations back to image object
-			attachment.focusPoint.x = percentageX;
-			attachment.focusPoint.y = percentageY;
+			if (attachment.focusPoint.y < 0) {
+				attachment.focusPoint.y = 0;
+			} else {
```

Comment:
it is elseif, but coding standards were f*cked up. It breaks it up to `else{ if { } }`. Changed my coding standards